src/test/java/com/empowerops/antextfx/TestLexicalBracketCountingHighlighterFixture.kt

```diff
@@ -8,10 +8,9 @@ import org.junit.Test
 /**
  * Created by Geoff on 4/11/2016.
  */
-class LexicalBracketCountingHighlighterFixture {
+class TestLexicalBracketCountingHighlighterFixture {
```

Comment:
In fact it [is a JUnit convention](http://junit.org/junit4/faq.html#running_15)! :)